lib/app.js

```diff
@@ -102,6 +102,7 @@ module.exports = settings => {
         return applicants;
       })
       .then(applicants => Profile.query().findByIds(applicants))
+      .then(profiles => profiles.filter(p => !p.asruUser))
```

Comment:
could this not just be a where clause on the query above?

Reply:
Maybe. I wasn't sure how well that played with finding by id, and I figured I'd have to find a way to actually run and test it if I did that. My way definitely works.